app/Http/Controllers/Api/V1/TeamController.php

```diff
@@ -482,13 +494,20 @@ public function addProjects(TeamAddProjectRequest $addProjectRequest, Team $team
                 // $team->projects()->attach($project);
                 // $assigned_projects[] = $project;
                 // pushed cloning of project in background
-                CloneProject::dispatch($auth_user, $project,  $addProjectRequest->bearerToken(), $team->organization->id, $team)->delay(now()->addSecond());
+                CloneProject::dispatch($auth_user,
```

Comment:
Please revert it back to previous, if it makes horizontal scroll then only we use break line